README.md

```diff
@@ -4,12 +4,21 @@
 [![Build Status](https://ci.appveyor.com/api/projects/status/fuaftu9mvp0y488j/branch/master?svg=true)](https://ci.appveyor.com/project/cyrush/blt/branch/master)
 [![Documentation Status](https://readthedocs.org/projects/llnl-blt/badge/?version=latest)](https://llnl-blt.readthedocs.io/en/latest/?badge=latest)
 
-A streamlined CMake build system foundation for developing HPC software
+BLT is a streamlined `CMake <https://cmake.org/>`_-based foundation for building, linking and testing HPC software.
 
-User Tutorial
----------------------
+BLT makes is easy to get up and running on a wide range of compilers (gcc, clang, intel, XL, Visual Studio), 
```

Comment:
I wanted to add a bit of context here about `blt` and what it is for.

Please update/improve/work on this more.

Reply:
maybe a bulleted list for each instead of parenthesis list?

* Compilers: gcc, clang, intel,...
* Operating Systems: OSX, Linux, Windows

Reply:
Thanks @white238 -- do you want to add some content/wording from your recent slides?

Reply:
I also think bullets would read a lot easier.

Reply:
Looks fine to me. Thanks kenny

Reply:
ok -- If we're going to bullets, I'll also add a line about built-in testing, benchmarking, documentation and code health frameworks.